apps/server/src/modules/learnroom/controller/dto/course.query.params.ts

```diff
@@ -12,4 +12,11 @@ export class CourseQueryParams {
 		enum: CommonCartridgeVersion,
 	})
 	version!: CommonCartridgeVersion;
+
+	@IsString()
+	@IsOptional()
+	@ApiPropertyOptional({
+		description: 'The ids of topics which should be exported separated by a comma.',
```

Comment:
What is the default behaviour when no topics are send? Will all topics be exported?